adapters/flipp/flipp.go

```diff
@@ -223,3 +224,35 @@ func buildBid(decision *InlineModel, impId string) *openrtb2.Bid {
 	}
 	return bid
 }
+
+func paramsUserKeyPermitted(request *openrtb2.BidRequest) bool {
+	if request.Regs != nil {
+		if request.Regs.COPPA == 1 {
+			return false
+		}
+		if request.Regs.GDPR != nil && *request.Regs.GDPR == 1 {
+			return false
+		}
+	}
+	if request.Ext != nil {
+		var extData struct {
+			TransmitEids bool `json:"transmitEids"`
+		}
+		if err := json.Unmarshal(request.Ext, &extData); err == nil {
+			if !extData.TransmitEids {
+				return false
+			}
+		}
+	}
+	if request.User != nil && request.User.Consent != "" {
+		tcModel, err := iabtcfv2.Decode(request.User.Consent)
+		if err != nil {
+			fmt.Printf("%v", err)
```

Comment:
Seems like a debug leftover. Should not add `fmt` line here

Reply:
removed